seed/postgres/config.go

```diff
@@ -0,0 +1,25 @@
+package postgres
+
+import "database/sql"
+
+// SeedConfig represents the configuration for the Postgres seed component.
+type SeedConfig struct {
+	// ClientProvider - Provides a postgres client to use.
+	// available only via code, not available in config files.
+	ClientProvider func() (*sql.DB, error) `json:"-"`
+
+	// Setup - a string that contains the SQL setup script to run before seeding the data.
+	Setup string `json:"-"`
```

Comment:
Should have json tag

Reply:
added. where is it being used?

Reply:
Envite can be used as a cli tool using yaml files...this allows parsing external yaml/json